mungegithub/mungers/matchers/comment/command_test.go

```diff
@@ -25,51 +25,66 @@ import (
-		expectedCommand *Command
-		comment         string
+		expectedCommands []*Command
+		comment          string
 	}{
 		{
-			expectedCommand: nil,
-			comment:         "I have nothing to do with a command",
+			expectedCommands: []*Command{},
+			comment:          "I have nothing to do with a command",
 		},
 		{
-			expectedCommand: nil,
-			comment:         " /COMMAND Line can't start with spaces",
+			expectedCommands: []*Command{},
+			comment:          " /COMMAND Line can't start with spaces",
 		},
 		{
-			expectedCommand: nil,
-			comment:         "Command not at the beginning:\n/COMMAND\nAnd something else...",
+			expectedCommands: []*Command{&Command{Name: "COMMAND"}},
+			comment:          "/COMMAND",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND"},
-			comment:         "/COMMAND",
+			expectedCommands: []*Command{&Command{Name: "COMMAND"}},
+			comment:          "/COMMAND\r",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND", Arguments: "Valid command"},
-			comment:         "/COMMAND Valid command",
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Args after tab"}},
+			comment:          "/COMMAND\tArgs after tab",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND", Arguments: "Multiple Lines"},
-			comment:         "/COMMAND Multiple Lines\nAnd something else...",
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Removes trailing backslash R"}},
+			comment:          "/COMMAND Removes trailing backslash R\r\n",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND", Arguments: "Args"},
-			comment:         "/COMMAND Args\n/OTHERCOMMAND OtherArgs",
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Valid command"}},
+			comment:          "/COMMAND Valid command",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND", Arguments: "Command name is upper-cased"},
-			comment:         "/command Command name is upper-cased",
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Multiple Lines"}},
+			comment:          "/COMMAND Multiple Lines\nAnd something else...",
 		},
 		{
-			expectedCommand: &Command{Name: "COMMAND", Arguments: "Arguments is trimmed"},
-			comment:         "/COMMAND     Arguments is trimmed   ",
+			expectedCommands: []*Command{
+				&Command{Name: "COMMAND", Arguments: "Args"},
+				&Command{Name: "OTHERCOMMAND", Arguments: "OtherArgs"},
+			},
+			comment: "/COMMAND Args\n/OTHERCOMMAND OtherArgs",
+		},
+		{
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Command name is upper-cased"}},
+			comment:          "/command Command name is upper-cased",
+		},
+		{
+			expectedCommands: []*Command{&Command{Name: "COMMAND", Arguments: "Arguments is trimmed"}},
+			comment:          "/COMMAND     Arguments is trimmed   ",
+		},
+		{
+			expectedCommands: []*Command{&Command{Name: "COMMAND"}},
+			comment:          "Command not at the beginning:\n/COMMAND\nAnd something else...",
 		},
```

Comment:
This checks that we can find commands not at the beginning of the comments but still at the beginning of the line.  Maybe add a test that verifies we ignore commands midway through the line.  And something that shows we ignore two forward slashes entirely (the comment thing)